### PR TITLE
populate frequently used emojis from the service [HOTPOT-130]

### PR DESCRIPTION
**Pull request by joshblum**
use the service backed `topEmojis` to populate the frequently used emojis on desktop and mobile

cc @keybase/hotpotatosquad

### CODE REVIEW

File: shared/chat/conversation/messages/react-button/emoji-picker/container.tsx [hunk 1]
```diff
@@ -26,7 +28,9 @@ type WrapperState = {
   width: number
 }
 
-const mapStateToProps = () => ({})
+const mapStateToProps = (state, ownProps: OwnProps) => ({
+  topReacjis: state.config.userReacjis ? state.config.userReacjis.topReacjis : defaultTopReacjis,
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/emoji-picker/container.tsx]: just default the store value to this

File: shared/chat/conversation/messages/react-button/emoji-picker/index.tsx [hunk 2]
```diff
@@ -7,11 +7,22 @@ import {chunk} from 'lodash-es'
 import {memoize} from '../../../../../util/memoize'
 
 // defer loading this until we need to, very expensive
-const getData = memoize(() => {
+const getData = memoize((topReacjis: Array<string>) => {
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/react-button/emoji-picker/index.tsx]: i don't think you want to memoize these two things together. keep them as 2 different things

File: shared/reducers/config.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
+import {store} from 'emoji-mart'
```
Comment (chrisnojima) [inline shared/reducers/config.tsx]: i don't think this belongs in the reducer. better as a side effect. also does emoji-mart work on mobile?

Comment (joshblum) [reply shared/reducers/config.tsx]: nope will short circuit on mobile

**Comment by joshblum** [comment]
@chrisnojima this is ready

File: shared/actions/config/index.tsx [hunk 4]
```diff
@@ -500,6 +502,37 @@ function* criticalOutOfDateCheck() {
   }
 }
 
+// onUpdateUserReacjis hooks `userReacjis`, frequently used reactions
```
Comment (chrisnojima) [inline shared/actions/config/index.tsx]: actually maybe this whole thing should be in chat and not config. We get a static chat config that maybe should hold this data

Comment (chrisnojima) [reply shared/actions/config/index.tsx]: cc: @mmaxim

Comment (joshblum) [reply shared/actions/config/index.tsx]: moved `userReacjis` over to the chat store, @mmaxim said the static config isn't the right place so chained to the bootstrap action

**Pull request merged**